src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -664,27 +672,31 @@
             default { "" }
             disablewhen "{ include_adaptor_wheels == 0 }"
             parmtag { "script_callback_language" "python" }
+            help    "The directory for adaptor wheels"
         }
         parm {
              name    "auto_unlock_rops"
              label   "Automatically unlock ROPs"
              type    toggle
              default { "0" }
              parmtag { "script_callback_language" "python" }
+             help    "A locked ROP will use existing outputs and won't re-render. It will also block dependencies from re-rendering. Select this option to automatically unlock dependency ROPs"
          }
          parm {
              name    "auto_parse_hip"
              label   "Automatically parse Hip references"
```

Comment:
```suggestion
             label   "Automatically parse hip references"
```

Following the same sentence-casing

Reply:
Updated to follow the same format as the Houdini docs: `scene (.hip) file`